tests/integration/ceph_upgrade_test.go

```diff
@@ -167,7 +167,8 @@ func (s *UpgradeSuite) TestUpgradeToMaster() {
 	require.NoError(s.T(), err)
 
 	osdsNotOldVersion := fmt.Sprintf("app=rook-ceph-osd,ceph-version!=%s", oldCephVersion)
-	err = s.k8sh.WaitForDeploymentCount(osdsNotOldVersion, s.namespace, numOSDs)
+	// It can take a LONG time to update the mgr modules, so wait an extra long time here
+	err = s.k8sh.WaitForDeploymentCountWithRetries(osdsNotOldVersion, s.namespace, numOSDs, utils.RetryLoop*6)
```

Comment:
The default is to wait 5 minutes in the loop, right? Now we're going to wait up to 30 minutes? Can we limit it to double the timeout? At some point it's not worth waiting so long.